browse/templates/user_banner.html

```diff
@@ -52,9 +50,9 @@ <h2>Happy Giving Tuesday - support arXiv today!</h2>
   {%- set now=request_datetime.strftime("%Y%m%d%H%M")|int -%}
   {%- if (SLIDE and (BANNER_START_1|int <= now) and (BANNER_END_1|int > now)) -%}
     {# only run this script while a banner is active #}
-    <link rel="stylesheet" type="text/css" media="screen" href="{{ url_for('static', filename='css/slider.css') }}?v=1.15" />
+    <link rel="stylesheet" type="text/css" media="screen" href="{{ url_for('static', filename='css/slider.css') }}?v=20241211" />
     <script src="//code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
-    <script type="text/javascript" src="{{ url_for('static', filename='js/donate.js') }}?v=1.11"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/donate.js') }}?v=121124"></script>
```

Comment:
Are the version identifiers really so different for donate.js and slider.css?

Why is the privacy banner including donate.js?

Why is the styling on the privacy banner "donate-cta" ?

Reply:
usually its a donate banner, for this particular one its repurposed to a privacy policy banner. The state of the css makes it not worth trying to rename everything depending on the latest banners purpose. I switched the versioning to dates for consistency since its hard to track if someone actually bothers to update the version number or not. The only purpose is providing a different parameter in the url to bypass cacheing